halo2_gadgets/src/utilities/lookup_range_check.rs

```diff
@@ -478,295 +705,380 @@ mod tests {
     };
     use pasta_curves::pallas;
 
+    use crate::{
+        sinsemilla::primitives::K,
+        tests::test_utils::test_against_stored_circuit,
+        utilities::lookup_range_check::{
+            LookupRangeCheck, PallasLookupRangeCheck, PallasLookupRangeCheck45BConfig,
+            PallasLookupRangeCheckConfig,
+        },
+    };
     use std::{convert::TryInto, marker::PhantomData};
 
```

Comment:
can we restructure to reduce diff size?

Reply:
The diff between this PR and [PR to add a LookupRangeCheck trait](https://github.com/zcash/halo2/pull/823) is quite small